tests/checks/integration/test_kong.py

```diff
@@ -0,0 +1,43 @@
+# stdlib
+import unittest
+
+# 3p
+from nose.plugins.attrib import attr
+
+# project
+from tests.checks.common import load_check
+
+
+@attr(requires='kong')
+class TestKong(unittest.TestCase):
```

Comment:
We actually created a class for the agent checks tests : https://github.com/DataDog/dd-agent/blob/master/tests/checks/common.py#L131

It should be easier to test and allows us to check the metrics coverage. (whether the agent receives all the metrics it should receive or not)

For a simple example : https://github.com/DataDog/dd-agent/blob/master/tests/checks/integration/test_activemq_xml.py
A more detailed one : https://github.com/DataDog/dd-agent/blob/master/tests/checks/integration/test_fluentd.py